src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -109,6 +124,8 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 			});
 		} else {
 			let tempWidgets = this.dashboardService.getSettings<Array<WidgetConfig>>([this.context, 'widgets'].join('.'));
+			// remove tasks widget because those will be shown in the toolbar
+			tempWidgets.shift();
```

Comment:
temp widgets are all widgets or only tasks?

Reply:
temp widgets are all the widgets. I updated it to make sure it's the tasks-widget being removed